### PR TITLE
LPS-140176  [CP2.0]  Automated test Add CP2DeveloperKeys

### PR DESCRIPTION
**Pull request by styller1**
https://issues.liferay.com/browse/LPS-146179

### CODE REVIEW

**Comment by styller1** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeveloperKeys.testcase [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+			CustomerPortal.enterOnProject(title = "Test Account LXC - SM");
+		}
+
+        task ("Go to the product activation page") {   
+            CP2NavigationMenu.goToProductActivation();
+
+		    CP2NavigationMenu.clickOnProductButton(product = "${testProduct}");
+        }
+
+		task ("Open the dropdown and assert if have 5 versions") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeveloperKeys.testcase:61]: Hi @styller1, I think that this task can be "Open the dropdown and verify that there are 5 DXP versions"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeveloperKeys.testcase [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+		    CP2NavigationMenu.clickOnProductButton(product = "${testProduct}");
+        }
+
+        task ("Click in the link 'deploying activation keys article' and go to the page") {
+            Click(locator1 = "CustomerPortalSite#DEPLOY_ARTICLE_PAGE");
+
+            Navigator.selectSpecificWindow(windowTitle = "Deploying Activation Keys – Liferay Help Center"); 
+        }
+
+        task("Assert the new page as open and go back to 'Product Activation page' and close the page 'Deploying Activation Keys'") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeveloperKeys.testcase]: Hi @styller1, I think that this task can be "Assert that a new page is opened and go back to 'Product Activation page'

**Review by AfonsoHSantos — APPROVED** [review]

**Comment by luanakarla** [comment]
ci:forward

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:forward

**Pull request closed without merging**